packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -559,6 +563,16 @@ static String interpretCircleOptions(Object o, CircleOptionsSink sink) {
     return points;
   }
 
+  private static List<List<LatLng>> toHoles(Object o) {
+    final List<?> data = toList(o);
+    final List<List<LatLng>> holes = new ArrayList<>(data.size());
+
+    for (Object ob : data) {
```

Comment:
maybe make the code more self explanatory by renaming the `ob` to something more meaningful?

Reply:
Done.

This style of naming is present in all methods in this class. So maybe it worth to refactor in separate PR.